js/classes.js

```diff
@@ -91,24 +147,119 @@ class Item {
         this.expenseMultipliers = [];
     }
 
+    pushDefaultMultipliers(){
+        //this.expenseMultipliers.push(getBoundTaskEffect('Bargaining'));
+        //this.expenseMultipliers.push(getBoundTaskEffect('Intimidation'));
+    }
+
     getEffect() {
         if (gameData.currentProperty !== this && !gameData.currentMisc.includes(this)) return 1;
-        return this.baseData.effect;
+        return this.baseData.effects;
     }
 
     getEffectDescription() {
-        let description = this.baseData.description;
-        if (itemCategories['Properties'].includes(this.name)) {
-            description = 'Population';
-        }
-        return 'x' + this.baseData.effect.toFixed(1) + ' ' + description;
+        return "not implemented";
+        //let description = this.baseData.description;
+        //if (itemCategories['Properties'].includes(this.name)) {
+        //    description = 'Population';
+        //}
+        //return 'x' + this.baseData.effects.toFixed(1) + ' ' + description;
     }
 
     getEnergyUsage() {
         return applyMultipliers(this.baseData.expense, this.expenseMultipliers);
     }
 }
 
+class Module{
+    #enabled = false;
+
+    constructor(baseData) {
+        this.data = baseData;
+        this.title = baseData.title;
+        this.components = baseData.components;
+    }
+
+    do(){
+        //Do whatever a module does
+    }
+
+    toggleEnabled(){
+        this.#enabled = !this.#enabled;
+        for (let component in this.components){
+            component.setEnabled(this.#enabled)
+        }
+        if (this.#enabled){
+            if (!gameData.currentModules.hasOwnProperty(this.name)){
+                gameData.currentModules[this.name] = this;
+            }
+        }
+        else{
+            if (gameData.currentModules.hasOwnProperty(this.name)) {
+                delete gameData.currentModules [this.name];
+            }
+        }
+    }
+}
+
+class ModuleComponent{

```

Comment:
```suggestion
class ModuleComponent {

```
check mal bitte insgesamt dein Formatting bzw dass der IntelliJ Formatter angewendet wird